Anabaena/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      language="javascript"
+      type="text/javascript"
+      src="../libraries/p5.min.js"
+    ></script>
+    <script type="module">
+      import { sketch } from "./Anabaena.js";
+      import "../sketchlib/sketchbook-navbar.js";
+      const p5_sketch = new p5(sketch, "sketch");
+    </script>
+    <link rel="stylesheet" href="../styles/layout.css" />
+  </head>
+  <body>
+    <sketchbook-navbar></sketchbook-navbar>
+    <div class="container">
+      <h1>Anabaena Catenula (2025)</h1>
+      <div id="sketch"></div>
+      <div class="description">
+        <p>
+          Move the mouse (desktop) or touch and drag the canvas (mobile) to move
+          the bacterium around.
+        </p>
```

Comment:
Add a description. I think it would be helpful to include the L-system grammar and a diagram (similar to the one in ABoP)

Reply:
Maybe this would be an opportunity to try MathJax?